submarine-cloud-v2/controller.go

```diff
@@ -1035,59 +1049,80 @@ func (c *Controller) syncHandler(key string) error {
 		return nil
 	}
 
-	// Get the Submarine resource with this namespace/name
-	submarine, err := c.submarinesLister.Submarines(namespace).Get(name)
-	if err != nil {
-		// The Submarine resource may no longer exist, in which case we stop
-		// processing
-		if errors.IsNotFound(err) {
-			utilruntime.HandleError(fmt.Errorf("submarine '%s' in work queue no longer exists", key))
-			return nil
+	klog.Info("syncHandler: ", key, " / ", action)
+
+	if action != "DELETE" { // Case: ADD & UPDATE
```

Comment:
Use const variable to replace "DELETE" ?

Reply:
@xunliu Thank you for your recommendation! I have changed this hardcoded string to constants.